<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditEventForm</class>
 <widget class="QWidget" name="EditEventForm">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>360</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>12</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>12</number>
   </property>
   <item>
    <widget class="QLineEdit" name="text_LineEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>21</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>21</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
     <property name="leftMargin">
      <number>12</number>
     </property>
     <property name="topMargin">
      <number>12</number>
     </property>
     <property name="rightMargin">
      <number>12</number>
     </property>
     <property name="horizontalSpacing">
      <number>4</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="5" column="0">
      <widget class="QLabel" name="color_Label">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>32</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">padding-bottom:2px</string>
       </property>
       <property name="text">
        <string>Color:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="endrepeat_Label">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>32</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">padding-bottom:2px</string>
       </property>
       <property name="text">
        <string>End Repeat:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="starts_Label">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>32</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">padding-bottom:2px</string>
       </property>
       <property name="text">
        <string>Starts:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="repeat_Label">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>32</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">padding-bottom:2px</string>
       </property>
       <property name="text">
        <string>Repeat:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="allday_Label">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>32</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">padding-bottom:2px</string>
       </property>
       <property name="text">
        <string>All day:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="alert_Label">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>32</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">padding-bottom:2px</string>
       </property>
       <property name="text">
        <string>Alert:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="ends_Label">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>32</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">padding-bottom:2px</string>
       </property>
       <property name="text">
        <string>Ends:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QDateTimeEdit" name="starts_DateTimeEdit">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>21</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="start_DateEdit">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>21</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QDateTimeEdit" name="finish_DateTimeEdit">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>21</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="finish_DateEdit">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>21</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QComboBox" name="endrepeat_ComboBox">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>21</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Never</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>On Date</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="endrepeat_DateEdit">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>21</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="endrepeat_horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="7" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QComboBox" name="sound_ComboBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>21</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sound_PushButton">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>21</height>
          </size>
         </property>
         <property name="palette">
          <palette>
           <active>
            <colorrole role="ButtonText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>249</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="ButtonText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>249</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="ButtonText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>31</red>
               <green>31</green>
               <blue>31</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="text">
          <string>▶</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="sound_Label">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>32</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">padding-bottom:5px</string>
       </property>
       <property name="text">
        <string>Sound:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QComboBox" name="alert_ComboBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>21</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QComboBox" name="color_ComboBox">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>21</height>
          </size>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="colorshow_Label">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QComboBox" name="repeat_ComboBox">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>21</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QCheckBox" name="allday_CheckBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0,0,0">
     <property name="spacing">
      <number>12</number>
     </property>
     <item>
      <widget class="QPushButton" name="delete_PushButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>38</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>38</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>31</red>
             <green>31</green>
             <blue>31</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>32</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_PushButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_PushButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>text_LineEdit</tabstop>
  <tabstop>allday_CheckBox</tabstop>
  <tabstop>starts_DateTimeEdit</tabstop>
  <tabstop>start_DateEdit</tabstop>
  <tabstop>finish_DateTimeEdit</tabstop>
  <tabstop>finish_DateEdit</tabstop>
  <tabstop>repeat_ComboBox</tabstop>
  <tabstop>endrepeat_ComboBox</tabstop>
  <tabstop>endrepeat_DateEdit</tabstop>
  <tabstop>color_ComboBox</tabstop>
  <tabstop>alert_ComboBox</tabstop>
  <tabstop>sound_ComboBox</tabstop>
  <tabstop>sound_PushButton</tabstop>
  <tabstop>save_PushButton</tabstop>
  <tabstop>cancel_PushButton</tabstop>
  <tabstop>delete_PushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
